plugins/jobs/src/js/types/JobLink.ts

```diff
@@ -0,0 +1,9 @@
+export const JobLinkSchema = `
+type JobLink {
```

Comment:
We need job links as we only want to return the jobId, right?

Reply:
I find this better than just say it returns a string. Maybe there is a type alias on graphql. But I would rather prefer the more verbose type now. 

If there is a way to alias a type, I can revisit that on the ticket I will focus on graphql.

Reply:
Yeah, me too, just wanted to be sure I understand what is going on there